compiler/haskell-ide-core/src/Development/IDE/State/Rules.hs

```diff
@@ -241,7 +241,7 @@ reportImportCyclesRule =
     where cycleErrorInFile f (PartOfCycle imp fs)
             | f `elem` fs = Just (imp, fs)
           cycleErrorInFile _ _ = Nothing
-          toDiag imp mods = set dLocation (Just loc) $ Diagnostic
+          toDiag imp mods = set dLocation loc $ ("",) $ Diagnostic
```

Comment:
Is this right? Looks like we're taking a src span which has a FIlePath, turning it into a URI, and back again.